### PR TITLE
Upgraded to pydetector 0.11.2. Updated integration tests

### LINKED ISSUE #57: status: ERROR errors: "column out of bounds: 0 [1, 1]"
Hi,
I tried to extract UAST from python file and got an unexpected result - result of extraction starts with the 
error message.

The content of `exp.py`:
```
# Here you can find the code of /app/exp.py:
# find x in range (-2, 2) that should minimize math.sin function
import math
from hyperopt import fmin, tpe, hp
from hyperopt.mongoexp import MongoTrials

mongodb = "mongodb"
db_port = "27017"
db_name = "ml_exp"
exp_key = "exp1"

trials = MongoTrials("mongo://%s:%s/%s/jobs" % (mongodb, db_port, db_name), exp_key="exp1")
best = fmin(math.sin, hp.uniform("x", -2, 2), trials=trials, algo=tpe.suggest, max_evals=10)
print("Result:", best)
```

and result of extraction:
```
status: ERROR
errors: "column out of bounds: 0 [1, 1]"
uast {
  internal_type: "Module"
...
```
Full UAST is attached: 
[exp.uast.txt](https://github.com/bblfsh/python-driver/files/1208902/exp.uast.txt)

PS:
is it correct that we have several lines of roles?
```
      roles: STRING_LITERAL
      roles: EXPRESSION
      roles: ASSIGNMENT_VALUE
```

### PR DESCRIPTION
**Pull request by juanjux**
Bump version of pydetector to get bblfsh/python-pydetector/pull/17. Regenerate integration tests. 

Fixes #57

### CODE REVIEW

File: tests/functiondef_docstring.py.uast [hunk 1]
```diff
@@ -1,5 +1,6 @@
-Status:  ok
+Status:  error
 Errors: 
+ .  column out of bounds: 0 [1, 8]
```
Comment (abeaumont) [inline tests/functiondef_docstring.py.uast]: This probably reflects a similar problem to the one that's being fixed.

File: tests/string_fstring.py.uast [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Status:  error
 Errors: 
- .  column out of bounds: 0 [1, 1]
+ .  column out of bounds: 2 [1, 1]
```
Comment (abeaumont) [inline tests/string_fstring.py.uast]: Same as above

File: tests/string_triple.py.uast [hunk 3]
```diff
@@ -1,5 +1,6 @@
-Status:  ok
+Status:  error
 Errors: 
+ .  column out of bounds: 0 [1, 4]
```
Comment (abeaumont) [inline tests/string_triple.py.uast]: same as above

File: tests/functiondef_docstring.py.uast (hunk 1 above)
Comment (juanjux) [inline tests/functiondef_docstring.py.uast]: fuuuuuuuuuuuuuuuuu....!

Yes, found another instance of the same problem for other node types, I will try with a more generic solution with the fix for #58 so I don't have two release two versions more of pydetector.

**Comment by juanjux** [comment]
Closing, will open a new PR for pydetector 0.11.3 with fixes for this and python-driver/issues/58

**Pull request closed without merging**